cmd/tetra/debug/progs.go

```diff
@@ -331,6 +332,9 @@ func getTetragonProgs(base string) ([]*prog, error) {
 			if finfo.IsDir() {
 				return nil
 			}
+			if strings.HasSuffix(path, "link") {
```

Comment:
Maybe Having "/link" is safer?
(suggestion based on observing what exists under /sys/fs/bpf/tetragon)

Reply:
right it is.. also we need to add '/link_override' .. it's not failing on it, but to make the change consistent